### PR TITLE
[FOLSPRINGB-61] : mod-spring-template: add JPA entities examples

### PR DESCRIPTION
**Pull request by arin-epm**
[FOLSPRINGB-61](https://issues.folio.org/browse/FOLSPRINGB-61)

## Purpose
1.Example on how to add JPA entities to the project and use Spring Data to access the database. 
2.Note that DB objects should be created by Liquibase and should not be created automatically by the Spring Data/Hibernate; there is a dedicated configuration parameter for that. 
3. Provide the link to the Spring Data project documentation.

## Approach
Adding the required documentation in README.md file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,11 +35,47 @@ This guide shows how to create a new folio backend module using the mod-spring-t
...
+    ID: Type of the id of the entity that repository manages (Generally the wrapper class of your @Id that is created inside the Entity/Model class)
+
+    For more information, follow:
+    * [Spring Data JPA - Reference Documentation](https://docs.spring.io/spring-data/jpa/docs/current/reference/html/)
+    * [Accessing Data with JPA](https://spring.io/guides/gs/accessing-data-jpa/)
+11. DB objects(tables/columns) should be created by Liquibase and should not be created automatically by the Spring Data/Hibernate thats why in your configuration file (application.properties/application.yml), set this property to "none":
+    ```yaml
+    spring:
+      jpa:
+        hibarnate:
```
Comment (TarasSpashchenko) [inline README.md]: hibernate not hibarnate. please fix

Comment (arin-epm) [reply README.md]: done

**Review by nirmalr-shah — APPROVED** [review]

**Comment by dcrossleyau** [comment]
That api-schema-lint error should be addressed. If it remains, then everyone who uses this template will get it always.

**Comment by NikitaSedyx** [comment]
I believe everyone who already uses it already gets link error, @arin-epm could you please fix it? just 2 lines to populate description

**Comment by arin-epm** [comment]
@NikitaSedyx issue fixed.

**Comment by dcrossleyau** [comment]
However, they are keys and values of "Parameters" not "Properties".

**Review by TarasSpashchenko — APPROVED** [review]

**Review by Arghya22 — APPROVED** [review]

File: src/main/resources/swagger.api/schemas/parameter.json [hunk 2]
```diff
@@ -1,13 +1,13 @@
 {
   "description": "List of key/value parameters of an error",
   "type": "object",
-  "properties": {
+  "parameters": {
```
Comment (dcrossleyau) [inline src/main/resources/swagger.api/schemas/parameter.json]: It was just each "description" that needed to change. Not the actual JSON property.

**Review by dcrossleyau — APPROVED** [review]
My review only applies to the missing schema descriptions.

**Pull request merged**